tools/@aws-cdk/ubergen/bin/ubergen.ts

```diff
@@ -545,4 +564,18 @@ function sortObject<T>(obj: Record<string, T>): Record<string, T> {
  */
 function unixPath(x: string) {
   return x.replace(/\\/g, '/');
+}
+
+/**
+ * Resolves the directory where we're going to collect all the libraries.
+ *
+ * By default, this is purposely the same as the monopackage root so that our
+ * two import styles resolve to the same files but it can be overridden by
+ * seeting `ubergen.libRoot` in the package.json of the uber package.
+ *
+ * @param uberPackageJson package.json contents of the uber package
+ * @returns The directory where we should collect all the libraries.
+ */
+function resolveLibRoot(uberPackageJson: PackageJson): string {
+  return uberPackageJson.ubergen?.libRoot ?? MONOPACKAGE_ROOT;
```

Comment:
Does `libRoot` need a `path.resolve()` to be safe?

Reply:
LOL did you hear me type this comment? 🤣

Reply:
LOL telepathically